ztor/src/main/kotlin/com/zenmo/ztor/plugins/Databases.kt

```diff
@@ -65,14 +65,14 @@ fun Application.configureDatabases(): Database {
 
             val repository = SurveyRepository(db)
 
-            val project = call.request.queryParameters.get("project")
-            if (project != null) {
-                val surveys = repository.getSurveysByProjectWithUserAccessCheck(project, userId)
-                call.respond(HttpStatusCode.OK, surveys)
-                return@get
-            }
+            val active = call.request.queryParameters["active"]?.toBoolean()
```

Comment:
same questions as before.. is available to filter by 'active' from the UI? or what's the use case?

Reply:
See answer above.